clang/lib/Headers/__clang_cuda_intrinsics.h

```diff
@@ -479,7 +479,291 @@ inline __device__ unsigned __funnelshift_rc(unsigned low32, unsigned high32,
   return ret;
 }
 
-#endif // !defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 320
+#pragma push_macro("__INTRINSIC_LOAD")
+#define __INTRINSIC_LOAD(__FnName, __AsmOp, __DeclType, __TmpType, __AsmType, \
+                         __Clobber)                                           \
+  inline __device__ __DeclType __FnName(const __DeclType *ptr) {              \
```

Comment:
`ptr`, and local variables should probably be prefixed, too. Nothing stops the users from passing `-Dptr=1`